webapp/src/App.tsx

```diff
@@ -30,6 +32,19 @@ const SoundEffectTWO = () => {
   );
 };
 
+const client = createClient({
+  url: import.meta.env.VITE_HASURA_GRAPHQL_ENDPOINT + '/v1/graphql',
+  fetchOptions: () => {
+    const token = import.meta.env.VITE_HASURA_GRAPHQL_ADMIN_SECRET;
```

Comment:
Just as an FYI we would _not_ want to use this in a production setting, because then anyone could perform any action in the DB, instead we'd make those queries when we log in _and_ allow unauthenticated queries for the public info by changing Hasura settings.

I'll have to check my calendar but I could walk you through an example of that if it works out.

Again fine for this demo where we have the "make it work" attitude